notification-service/src/main/java/com/lms/notificationservice/database/NotificationDatabaseController.java

```diff
@@ -0,0 +1,103 @@
+package com.lms.notificationservice.database;
+
+import com.lms.notificationservice.model.Notification;
+import com.mysql.cj.jdbc.AbandonedConnectionCleanupThread;
+import jakarta.persistence.*;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.sql.*;
+import java.time.LocalDateTime;
+import java.util.*;
+import java.util.logging.Logger;
+
```

Comment:
Can add javadoc here just quick brief on class and u as @author, not mandatory for now can do later